README.md

```diff
@@ -265,11 +267,12 @@ To see the full-color output of a specific integration test, you can use `PRINT_
 
 ## Releasing a new version (Shopify employees)
 
-1. Update the version number in `version.rb` and commit that change alone to master.
-2. Tag the version with `git tag vx.y.z && git push --tags`
-3. Use the [Shipit Stack](https://shipit.shopify.io/shopify/kubernetes-deploy/rubygems) to build the `.gem` file and upload to [rubygems.org](https://rubygems.org/gems/kubernetes-deploy).
+1. Update the version number in `version.rb` and commit that change with message "Version x.y.z". Don't push yet or you'll confuse Shipit.
+2. Tag the version with `git tag vx.y.z -a -m "Version x.y.z"`
```

Comment:
Sorry, I somehow missed the point when we stopped doing `rake release` and started using Shipit.

Looking at how Shipit [releases the gem](https://shipit.shopify.io/shopify/kubernetes-deploy/rubygems/deploys/464645), I see that it takes care of creating and pushing the tag. Why it it necessary to do it by hands as suggested in step 2 of the README?